README.md

```diff
@@ -1,6 +1,6 @@
 > 🚀 We are granting pilot access to **Ivy\'s Tracer and Transpiler**
 > to some users, [join the waitlist](https://console.unify.ai/) if you
-> want to test them out!
+> want to test them out follow the steps given below!

```

Comment:
The steps are not described "below", the primary call of action is for the user to visit the console